### PR TITLE
Adjust Resolve documentation (user+admin) for recent publisher work.

### PR DESCRIPTION
**Pull request by robin-ynput**
## Changelog Description
As suggested on Discord, adjust `Resolve` documentation to reflect recent work:
* Ensure `Python` and `OpenTimelineIO` information is up to date
* Change instruction link from `openpype` to `ayon_resolve` for `Python 3.6`
* Rephrase and adjust visuals based on new publisher from `ayon-resolve-0.3.0`

## Testing notes:

1. Validated through a local docusaurus build

### CODE REVIEW

File: website/yarn.lock [hunk 1]
```diff
@@ -13590,4 +13590,4 @@ yocto-queue@^0.1.0:
 zwitch@^1.0.0:
   version "1.0.5"
   resolved "https://registry.npmjs.org/zwitch/-/zwitch-1.0.5.tgz"
-  integrity sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==
+  integrity sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==
```
Comment (robin-ynput) [inline website/yarn.lock:13593]: Im not sure where this is coming from.. local docusaurus build ?

File: website/docs/addon_resolve_artist.md [hunk 2]
```diff
@@ -114,7 +114,9 @@ After all clips which are intended to be converted to publishable instances are
 
 After the menu widget is opened (it can take while so be patient please :).
 
-Hit `Create ...` and then set **Use selection** to active and select the family to **Create Publishable Clips**.
+Hit `Create ...` and then select the family to **Create Publishable Clip**.
+
+You can leave `Use only clips with Chocolate clip colors` to publish only the selection.
```
Comment (BigRoy) [inline website/docs/addon_resolve_artist.md:119]: Did you mean to mention to leave it enabled?
```suggestion
You can leave `Use only clips with Chocolate clip colors` enabled to publish only the selection.
```

File: website/docs/addon_resolve_artist.md [hunk 3]
```diff
@@ -233,13 +235,14 @@ There are two ways to create a publishable editorial package: by creating a new
 
 ### Create/Convert a Timeline from Media Pool into an Editorial Package
 
-To create an editorial package, select a timeline in the media pool (1) and use the Create tool (2) to mark the timeline with Editorial Package (3) publishable product data. Keep _`Use selection`_ (5) active and choose _`Editorial Package`_ (3) as the product type. You can also use a Variant (4) to differentiate between different versions of timelines within the same target context. Hit Create (6).
+To create an editorial package, select a timeline in the media pool (1) and use the Create tool (2).
```
Comment (BigRoy) [inline website/docs/addon_resolve_artist.md:238]: Is it worth it here to clarify we're referring to the AYON Create?
```suggestion
To create an editorial package, select a timeline in the media pool (1) and use the AYON Create tool (2).
```

File: website/yarn.lock (hunk 1 above)
Comment (BigRoy) [reply website/yarn.lock:13593]: The only change is a lacking new line at the end of the file (maybe due to OS differences on how the tool generates the the line ending at the end of the file or whatever?)

**Review by BigRoy — APPROVED** [review]
Some small tiny notes - otherwise looks good to me.

And another question:
What does the 'in complience with AYON' mean? Is the launcher supposed to be built with 3.10 already? (it's 3.9 still for me). If AYON desktop tools are still 3.9 currently - what makes it 'comply with AYON'?

File: website/docs/addon_resolve_admin.md [hunk 4]
```diff
@@ -78,12 +79,17 @@ python3 -m pip install PySide6
...
 ### OpenTimelineIO
 
+
+AYON is using OpenTimelineIO for editorial publishing. OpenTimelineIO has to be installed into the Resolve Python environment.
+Tested versions:
+- Resolve 19 - `OpenColorIO 0.17.0` with `Python 3.10.x`
+- Resolve 18 - `OpenColorIO 0.17.0` with `Python 3.10.x`
+- Resolve 17 - `OpenColorIO 0.13.0` with `Python 3.6.2` (see special build instruction below)
+
 :::warning Permissions
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/addon_resolve_admin.md:89]: I wonder is the tip title is related to the tip. 
what about 
```suggestion
:::tip OpenColorIO 0.13.0
```

File: website/docs/addon_resolve_artist.md (hunk 2 above)
Comment (MustafaJafar) [reply website/docs/addon_resolve_artist.md:119]: Does the word `family` in this context is related to resolve or AYON?
should it be ? 
```
Hit `Create ...` and then select the publish type **Create Publishable Clip**.
```

File: website/yarn.lock (hunk 1 above)
Comment (MustafaJafar) [reply website/yarn.lock:13593]: I always running the docs live without building it.
Running live from your branch went fine

File: website/docs/addon_resolve_artist.md (hunk 2 above)
Comment (BigRoy) [reply website/docs/addon_resolve_artist.md:119]: It should be `product type` actually you are correct.

File: website/docs/addon_resolve_artist.md (hunk 3 above)
Comment (robin-ynput) [reply website/docs/addon_resolve_artist.md:238]: Looks good to me !

File: website/yarn.lock (hunk 1 above)
Comment (MustafaJafar) [reply website/yarn.lock:13593]: since this file is not related to this PR, could we revert the changes in it before merge ?

Comment (robin-ynput) [reply website/yarn.lock]: You got it !

**Pull request merged**